jenkins_jobs/modules/triggers.py

```diff
@@ -1232,6 +1232,7 @@ def github_pull_request(registry, xml_parent, data):
         matching, and must be separated by a new line. Exclusions take
         precedence over inclusions, if there is an overlap between included
         and excluded regions. (optional)
+    :arg str config-version: Config Version of the plugin. (optional)
```

Comment:
I don't think this should be configurable. It's part of the implementation. I mean JJB generates the config in version=3, so that when the plugin is parsing the data it knows how to process it. If you specify `config-version` to any other value then what will happen?